Dockerfile_build_ubuntu64_go19

```diff
@@ -21,7 +21,7 @@ RUN gem install fpm
 
 # Install go
 ENV GOPATH /root/go
-ENV GO_VERSION 1.9.1
+ENV GO_VERSION 1.9.2
```

Comment:
The _go19 file isn't really necessary - it should be _go110 but I'm not sure whether that's available yet. I'm fine leaving the file around so we don't have to undo and redo the work to have circle test the next Go version.

Reply:
Yeah.  I left it as is since I couldn't find a go1.10.